content/community-update/community-update/dcp-updates.md

```diff
@@ -6,6 +6,35 @@ title: DCP Updates
 
 # DCP Platform Updates
 
+#### September 26, 2022
+
+### Release Highlights
+
+* We've reached the 20th release of the DCP with data from over 30 million cells! Thank you to the HCA community and all of the contributors that helped us reach this milestone!
```

Comment:
We typically don't say data release - we could say data portal now hosts data for over 30 million cells.